ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyCategoriesSelectField.tsx

```diff
@@ -60,12 +76,14 @@ function PolicyCategoriesSelectField(): ReactElement {
                 selections={field.value}
                 onSelect={onSelectHandler(field.value)}
                 onToggle={onCategoriesToggle}
-                onCreateOption={onCreateCategory}
+                onCreateOption={
+                    isPolicyCategoriesEnabled ? () => undefined as void : onCreateCategory
+                }
```

Comment:
oh just curious why the `as void` is necessary here. can you not just do `() => {}` ?

Reply:
Good question. Copied from PatternFly docs.